### ISSUE TITLE
Integrate new accessibility artifact into Collab Cycle

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#47916]
## User Story
As a VFS team member, I want to be made aware of the new accessibility artifact for Staging Review so that I can adequately prepare my team and ensure I'm meeting the Platforms expectations.

As an accessibility specialist supporting the Collaboration Cycle Staging Review, I want to have clear guidance and steps to follow for when VFS teams submit the new accessibility artifact so that I can ensure I'm providing a consistent review to every VFS team/product at Staging Review. 

Assignee: @briandeconinck 
Peer Reviewer: @sterkenburgsara @allison0034 

## Description
In ticket #47915, Governance team created a new yaml template for VFS teams to complete when scheduling a Staging Review. In this ticket, we will update both operational documentation and VFS-facing guidance on Platform Website with regards to how the new artifact will be used.

In addition, we must determine if this new artifact will replace previous accessibility artifacts that VFS teams have been asked to use.

## Impacted Artifacts
VFS facing:
[PW/CC/Staging Review touchpoint page](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/staging-review)
[WCAG 2.1 success criteria and foundational testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing)
[Add accessibility test cases in TestRail](https://depo-platform-documentation.scrollhelp.site/developer-docs/add-accessibility-test-cases-in-testrail)
[Prepare for an accessibility staging review](https://depo-platform-documentation.scrollhelp.site/developer-docs/prepare-for-an-accessibility-staging-review)
[Staging Review gh template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=collab-cycle-review%2C+collaboration-cycle%2C+governance-team%2C+staging-review&template=staging-review.md&title=Staging+Review+%5BTeam+Name+-+Feature+Name%5D) 
[Staging Code block of Collab Cycle request ticket](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1749286928/Collaboration+Cycle+Kickoff+-+Moderator+Responsibilities#Staging-Review-code-block)
[Change history log](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/change-history) - draft new post

Operational:
[How to document feedback for Staging Review](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1839530098/How+to+document+feedback+for+Staging+Review)
[How to complete an Accessibility review for Staging review touchpoint](https://vfs.atlassian.net/wiki/spaces/PPS/pages/2059763776/How+to+complete+an+Accessibility+review+for+Staging+review+touchpoint)

## Tasks
- [x] Before beginning this work, answer the following questions below and review responses with Shira to ensure alignment:
1. What is the name of the new Accessibility artifact created in ticket #47915?
2. Will the new accessibility artifact be the only artifact we request from VFS teams for accessibility? (excluding other artifacts, such as Staging link, that the rest of the team also evaluates)
3. What happens if the VFS team doesn't submit the a11y artifact?
4. What happens if the VFS team doesn't submit a completed a11y artifact?
5. Should the a11y artifact be completed before or after the VFS team schedules staging review? aka within or outside of the 4 day window.
6. How can we reference the new a11y artifact in our final feedback? aka utilization of the DraftingTemplate
7. Will the new a11y artifact have a unique gh label? What other labels should it have?
8. Do we want the VFS team to just link to it in the artifacts section of the ticket or should they attach as a child ticket?
9. Will Gov team members communicate with the VFS team via the new a11y artifact? Or will it be just an artifact VFS submits and then all communicate is via the issue tickets?
- [x] Receive OCTO approval for making the artifact required
  - Note: Since we will likely ask for the new artifact to be a requirement, we will need to get OCTO approval at Platform Office hours. Nothing can be published until we receive OCTO approval.
- [x] Once all above questions are answered and there is team alignment, make draft changes of every impacted artifact and save all under Documentation Drafts subfolder -- [All drafts impacted by this ticket](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2431975472/47916+-+Integrate+new+accessibility+artifact+into+Collab+Cycle)
   - [x] VFS facing
      - [x] PW/CC/Staging Review touchpoint page
         - [Draft - Staging review](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2437840933/47916+-+Staging+review) (Changes in the Accessibility artifacts section)
      - [ ] ~WCAG 2.1 success criteria and foundational testing~
         - Proposing to archive this page. It's purely informational about changes included in WCAG 2.1. While that was timely information when WCAG 2.1 was adopted, it's less relevant now --- all changes should already be incorporated into the VA.gov experience standards, which is what we want VFS teams to be referencing anyway.
      - [x] Add accessibility test cases in TestRail
         - [Draft - Add accessibility test cases in TestRail](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2437873665/47916+-+Add+accessibility+test+cases+in+TestRail)
      - [x] Prepare for an accessibility staging review
         - [Draft - Prepare for an accessibility staging review](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2432204801/47916+-+Prepare+for+an+accessibility+staging+review)
      - [x] ~Staging Review gh template~
         - ~[Draft - staging-review.md](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2437742796/47916+-+staging-review.md)~
         - Resolved by #42846.
      - [x] Staging Code block of Collab Cycle request ticket
         - [Draft - Collaboration Cycle Kickoff - Moderator Responsibilities](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2437840957/47916+-+Collaboration+Cycle+Kickoff+-+Moderator+Responsibilities), staging review code block
      - [x] Change history log - draft new post
         - [Draft - Change history post](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2437513513/47916+-+Change+history+post)
   - [x] Operational
      - [ ] ~How to document feedback for Staging Review~
         - Nothing to update, Shira in agreement.
      - [x] How to complete an Accessibility review for Staging review touchpoint
         - [Draft - How to complete an Accessibility review for Staging review touchpoint](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2438692865/47916+-+How+to+complete+an+Accessibility+review+for+Staging+review+touchpoint)
             - Added an artifact review task to Step 1, and added a Step 11 for providing feedback on the artifact. Everything else should be the same.
- [x] Make updates to artifacts as needed and peer review
- [x] When peer review is complete, move forward with publishing to Platform Website.
   - [x] [New artifact yaml form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=briandeconinck&labels=a11y-testing&template=a11y-testing.yaml&title=Accessibility+Testing+for+%5BTeam+Name%2C+Product+Name%2C+Feature+Name%5D) created in va.gov-team repo
   - [x] Updated [Staging review](https://vfs.atlassian.net/wiki/spaces/CC/pages/1810137181/Staging+review)
   - [x] Updated [Add accessibility test cases in TestRail](https://vfs.atlassian.net/wiki/spaces/pilot/pages/1601307071/Add+accessibility+test+cases+in+TestRail)
   - [x] Updated [Prepare for an accessibility staging review](https://vfs.atlassian.net/wiki/spaces/CC/pages/1909293137/Prepare+for+an+accessibility+staging+review)
   - [x] Move Prepare for an accessibility staging review to *Collaboration cycle > Touchpoints > Staging review*
- [x] When peer review is complete, publish changes to internal documentation.
   - [x] Updated [Collaboration Cycle Kickoff - Moderator Responsibilities](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1749286928/Collaboration+Cycle+Kickoff+-+Moderator+Responsibilities)
   - [x] Updated [How to complete an Accessibility review for Staging review touchpoint](https://vfs.atlassian.net/wiki/spaces/PPS/pages/2059763776/How+to+complete+an+Accessibility+review+for+Staging+review+touchpoint)
- [x] When all modified artifacts are published to Platform Website
   - [x] Double-check links in artifact and in published pages, and fix as needed
   - [x] Publish Change log post 
- [x] Announce changes following the Platform Communication Playbook

## Peer Review
_To be completed by peer reviewer_
- [x] User story and acceptance criteria are met

## Acceptance Criteria
- [x] OCTO has approved of the new accessibility artifact and our request to make it a required artifact at Staging Review
- [x] VFS teams are notified of the new accessibility artifact
- [x] VFS teams understand that the new accessibility artifact is a requirement for Staging Reviews
- [x] Governance team is able to review completed accessibility artifacts at Staging Review

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by briandeconinck** [comment]
Attempt to answer the pre-work questions @shiragoodman.

1. What is the name of the new Accessibility artifact created in ticket #47915?
   - My proposal: Keep it simple and call it `Accessibility Testing`. That leaves it open-ended enough to add advanced testing into the artifact, and should be easy for VFS teams to find.
2. Will the new accessibility artifact be the only artifact we request from VFS teams for accessibility? (excluding other artifacts, such as Staging link, that the rest of the team also evaluates)
   - My proposal: Yes. We've got permission from OCTO-DE to make it required, and allowing them to submit other stuff just muddies the waters.
   - My only concern here is what happens to the [TestRail accessibility template](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc&display_deleted_cases=0). In addition to checklists (roughly) corresponding to foundational testing, there are checklists for subsets of WCAG success criteria too. Testing against those specific SCs goes above and beyond what we require (good!) but also duplicates some of what we require (less good). I'm not sure if it really provides value, but I also don't want to discourage teams from doing additional testing either. @shiragoodman thoughts?
3. What happens if the VFS team doesn't submit the a11y artifact?
   - My proposal: Prompt them to submit if not submitted 4 days beforehand, and if not provided at least 2 days beforehand then reschedule. Since it's a new process I don't want to cause too much pain for VFS teams, but if we're going to require it then it should be required. 
4. What happens if the VFS team doesn't submit a completed a11y artifact?
   - My proposal: Keep to the current schedule, Brian engages with the team prior to staging. If a team doesn't know how to do some portion of the testing, that should be an opportunity to educate and I want them to be upfront about it. But we can reevaluate if teams are regularly skipping tests. 
5. Should the a11y artifact be completed before or after the VFS team schedules staging review? aka within or outside of the 4 day window.
   - My proposal: The a11y artifact should be submitted on the same schedule as the rest of the artifacts, by 4 days before review. That keeps our artifact message to VFS teams consistent.
   - My hesitation here: Teams may theoretically be working on code up to the start of the 4 day window, so if they're also supposed to submit their a11y testing artifact then they'll potentially be doing a11y testing against code that's still changing. I don't know how much of an actual problem it will be (something to listen for from VFS teams), and flexibility on (4) above helps give teams some wiggle room.
6. How can we reference the new a11y artifact in our final feedback? aka utilization of the DraftingTemplate
   - My proposal: As an MVP, I would try to:
      - Reference anything relevant from the submitted artifact in my DraftingTemplate feedback.
      - Either comment on the artifact or edit the artifact (or both) to link to relevant feedback tickets after they're generated.
   - This is maybe the wishy-washiest part of this new artifact, but I think I'm okay with that? It'll take a few staging reviews to figure out how much things should be linked together and how useful that is for VFS teams.
7. Will the new a11y artifact have a unique gh label? What other labels should it have?
   - My proposal: Artifact gets the label `a11y-testing` and teams are instructed to add their team, product, and feature (if applicable) labels. I think that should give us enough to sort off of.
8. Do we want the VFS team to just link to it in the artifacts section of the ticket or should they attach as a child ticket?
   - My proposal: Just link. We don't have any other artifacts that are GH issues so we don't ask teams to add anything else as a child ticket. Linking is consistent with how we treat other artifacts and gives teams the flexibility to add it as a child to any other epics they might feel are relevant.
9. Will Gov team members communicate with the VFS team via the new a11y artifact? Or will it be just an artifact VFS submits and then all communicate is via the issue tickets?
   - My proposal: If I am **logging an issue with their product**, that happens in a feedback ticket generated from the DraftingTemplate. If a feedback ticket describes an issue they should have caught as part of their foundational testing, then I **communicate in the artifact that something got missed in their testing**. That communication will reference the feedback ticket and the conversation will be focused on the testing process -- what to do differently next time to catch this kind of thing -- and not how to fix that particular issue. I think we would want to make that distinction clear in the documentation that goes with the artifact too, although I'm sure the line will be blurry to some teams.

**Comment by briandeconinck** [comment]
@shiragoodman (and also @sterkenburgsara and @allison0034) A separate question that may fit into the scope of this ticket:

The [Prepare for an accessibility staging review](https://depo-platform-documentation.scrollhelp.site/developer-docs/prepare-for-an-accessibility-staging-review) page is going to need some pretty extensive edits, both to fit the new artifact and to clean up stuff that needs cleaning. While doing that, does it make sense to move the page?

Right now, it lives under [Developer docs > QA and accessibility testing > Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-testing). But the testing outlined in the artifact isn't necessarily something that engineers will be doing. As an example, in today's Authenticated Experience staging review, [Angela Agosto did most of the accessibility testing](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/payment-history/launch-materials/payment-info-v2-a11y-staging-review-prep.md).

I would propose moving the page to [Collaboration cycle > Touchpoints > Staging review](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/staging-review). That fits better with the intent of the page (you're getting ready for your staging review!) and is practice area agnostic/more likely for a PM to see and assign tasks to their team.

Thoughts?

I'll also add: I think multiple pages under Developer docs > QA and accessibility testing > Accessibility testing might need some attention, but that's outside the scope of this ticket. I'll work on writing up something for the backlog.

**Comment by shiragoodman** [comment]
@briandeconinck I am going to reply in line to the pre-work questions. 

I am 100% on board with moving the Prepare for an a11y staging review page over to the Collaboration Cycle space. Will we need to make edits to that page _before_ publishing the Accessibility Testing artifact? Or is it sufficient in its current state, meaning we can update it post the launch of the Accessibility Testing artifact?

Thank you for recognizing that the other a11y artifacts may need updating as well. Which specific docs are you referring to? What are your thoughts on their current location under Developer Docs? Here's my 2 cents:
1. I drafted [this ticket.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/49312) Please take a look and lmk your thoughts. 
2. I would like to move the[ Accessibility Testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-testing) up one level so it is no longer nested under "QA and accessibility testing" (meaning that space is now just renamed "QA testing". Should we also rename it to "Accessibility" and drop "testing" since the docs in that folder go beyond testing?

**Comment by shiragoodman** [comment]
> Attempt to answer the pre-work questions @shiragoodman.
> 
> 1. What is the name of the new Accessibility artifact created in ticket [Create new accessibility artifact #47915](https://github.com/department-of-veterans-affairs/va.gov-team/issues/47915)?
>    
>    * My proposal: Keep it simple and call it `Accessibility Testing`. That leaves it open-ended enough to add advanced testing into the artifact, and should be easy for VFS teams to find.

- Shira: I agree.

> 2. Will the new accessibility artifact be the only artifact we request from VFS teams for accessibility? (excluding other artifacts, such as Staging link, that the rest of the team also evaluates)
>    
>    * My proposal: Yes. We've got permission from OCTO-DE to make it required, and allowing them to submit other stuff just muddies the waters.
>    * My only concern here is what happens to the [TestRail accessibility template](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc&display_deleted_cases=0). In addition to checklists (roughly) corresponding to foundational testing, there are checklists for subsets of WCAG success criteria too. Testing against those specific SCs goes above and beyond what we require (good!) but also duplicates some of what we require (less good). I'm not sure if it really provides value, but I also don't want to discourage teams from doing additional testing either. @shiragoodman thoughts?
- Shira: I think we should remove the TestRail accessibility template from the Staging Review guidance as a requested artifact, however keep it listed as an optional artifact. We can keep it linked to the [Add accessibility test cases in TestRail](https://depo-platform-documentation.scrollhelp.site/developer-docs/add-accessibility-test-cases-in-testrail) guidance on Platform Website, with a caveat added that explains how this artifact differs from the Accessibility Testing artifact (pros, cons). If VFS teams still want to complete the TestRail tests, they can, and they'll still have guidance, but it won't be a requested artifact at Staging Review. 

> 3. What happens if the VFS team doesn't submit the a11y artifact?
>    
>    * My proposal: Prompt them to submit if not submitted 4 days beforehand, and if not provided at least 2 days beforehand then reschedule. Since it's a new process I don't want to cause too much pain for VFS teams, but if we're going to require it then it should be required.

- Shira: I like this. OCTO gave us permission to make it a required artifact, so I think if the VFS team doesn't submit, then no Staging Review. The warning at 4 days (like we do for all artifacts) is helpful, with a prompt to reschedule at 2 days (granted the Accessibility Artifact is the only one missing. If we don't have the staging link 4 days out, we'll reschedule).

> 4. What happens if the VFS team doesn't submit a completed a11y artifact?
>    
>    * My proposal: Keep to the current schedule, Brian engages with the team prior to staging. If a team doesn't know how to do some portion of the testing, that should be an opportunity to educate and I want them to be upfront about it. But we can reevaluate if teams are regularly skipping tests.

- Shira: sounds good to me. 

> 5. Should the a11y artifact be completed before or after the VFS team schedules staging review? aka within or outside of the 4 day window.
>    
>    * My proposal: The a11y artifact should be submitted on the same schedule as the rest of the artifacts, by 4 days before review. That keeps our artifact message to VFS teams consistent.
>    * My hesitation here: Teams may theoretically be working on code up to the start of the 4 day window, so if they're also supposed to submit their a11y testing artifact then they'll potentially be doing a11y testing against code that's still changing. I don't know how much of an actual problem it will be (something to listen for from VFS teams), and flexibility on (4) above helps give teams some wiggle room.

- Shira: this is the same concern with QA. I agree, they should complete their a11y testing once code is stable. We can watch it and see how teams handle this.

> 6. How can we reference the new a11y artifact in our final feedback? aka utilization of the DraftingTemplate
>    
>    * My proposal: As an MVP, I would try to:
>      
>      * Reference anything relevant from the submitted artifact in my DraftingTemplate feedback.
>      * Either comment on the artifact or edit the artifact (or both) to link to relevant feedback tickets after they're generated.
>    * This is maybe the wishy-washiest part of this new artifact, but I think I'm okay with that? It'll take a few staging reviews to figure out how much things should be linked together and how useful that is for VFS teams.

- Shira: My preference would be that you link to the Accessibility Testing artifact from the Drafting Template so that all the issue tickets we open appropriately reference the Accessibility Testing artifact. VFS teams will be able to see from the Accessibility Testing artifact which a11y-related issues were opened bc GitHub always notifies you when issues are cross-referenced. I took this screenshot from this ticket as an example:
![image](https://user-images.githubusercontent.com/71330581/200002418-98292e93-5369-4bd4-869c-06898a647969.png)
- I'd rather you not have to edit and/or comment in the new Accessibility Artifact post your review to connect the dots for the VFS team. However, in time, if we do notice VFS teams aren't "getting it" then we can come back to this and re-discuss.

> 7. Will the new a11y artifact have a unique gh label? What other labels should it have?
>    
>    * My proposal: Artifact gets the label `a11y-testing` and teams are instructed to add their team, product, and feature (if applicable) labels. I think that should give us enough to sort off of.

- Shira: sounds good.

> 8. Do we want the VFS team to just link to it in the artifacts section of the ticket or should they attach as a child ticket?
>    
>    * My proposal: Just link. We don't have any other artifacts that are GH issues so we don't ask teams to add anything else as a child ticket. Linking is consistent with how we treat other artifacts and gives teams the flexibility to add it as a child to any other epics they might feel are relevant.

- Shira: sounds good.

> 9. Will Gov team members communicate with the VFS team via the new a11y artifact? Or will it be just an artifact VFS submits and then all communicate is via the issue tickets?
>    
>    * My proposal: If I am **logging an issue with their product**, that happens in a feedback ticket generated from the DraftingTemplate. If a feedback ticket describes an issue they should have caught as part of their foundational testing, then I **communicate in the artifact that something got missed in their testing**. That communication will reference the feedback ticket and the conversation will be focused on the testing process -- what to do differently next time to catch this kind of thing -- and not how to fix that particular issue. I think we would want to make that distinction clear in the documentation that goes with the artifact too, although I'm sure the line will be blurry to some teams.

- Shira: I'm somewhat torn on this one. similar to #6 above, I would like all of your communication to the VFS team to take place in your issue tickets via the DraftingTemplate. However, if a VFS team does not execute the a11y test correctly, I suppose I am ok with you commenting in the Accessibility TEsting artifact "what to do differently next time to catch this kind of thing" instead of documenting that in the DraftingTemplate. Please keep an eye on this when launched and let me know. I worry it could become too burdensome to now have to document your fb on the issue and your fb on the testing in 2 different places.

**Comment by briandeconinck** [comment]
>Thank you for recognizing that the other a11y artifacts may need updating as well. Which specific docs are you referring to? What are your thoughts on their current location under Developer Docs?

The one that caught my eye was ["WCAG 2.1 success criteria and foundational testing"](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing), which has some really useful information but is written in the future tense and seems unlikely to be referenced by VFS teams at this point. The other pages are probably worth a closer read too but that's the big one.

>I drafted this ticket. Please take a look and lmk your thoughts.
>I would like to move the Accessibility Testing up one level so it is no longer nested under "QA and accessibility testing" (meaning that space is now just renamed "QA testing"). Should we also rename it to "Accessibility" and drop "testing" since the docs in that folder go beyond testing?

Made one minor edit to the Accessibility Standards page ticket, might look at it again when I'm back from PTO. Definitely agree on moving Accessibility Testing up a level and dropping "Testing" from the title.

> Shira: I think we should remove the TestRail accessibility template from the Staging Review guidance as a requested artifact, however keep it listed as an optional artifact. We can keep it linked to the Add accessibility test cases in TestRail guidance on Platform Website, with a caveat added that explains how this artifact differs from the Accessibility Testing artifact (pros, cons). If VFS teams still want to complete the TestRail tests, they can, and they'll still have guidance, but it won't be a requested artifact at Staging Review.

I'm good with this. The TestRail template can be a useful resource for teams but it's not a part of the collab cycle.

> Shira: My preference would be that you link to the Accessibility Testing artifact from the Drafting Template so that all the issue tickets we open appropriately reference the Accessibility Testing artifact. VFS teams will be able to see from the Accessibility Testing artifact which a11y-related issues were opened bc GitHub always notifies you when issues are cross-referenced. I took this screenshot from this ticket as an example:
> image
> I'd rather you not have to edit and/or comment in the new Accessibility Artifact post your review to connect the dots for the VFS team. However, in time, if we do notice VFS teams aren't "getting it" then we can come back to this and re-discuss.

I'm good with this!

> Shira: I'm somewhat torn on this one. similar to # 6 above, I would like all of your communication to the VFS team to take place in your issue tickets via the DraftingTemplate. However, if a VFS team does not execute the a11y test correctly, I suppose I am ok with you commenting in the Accessibility TEsting artifact "what to do differently next time to catch this kind of thing" instead of documenting that in the DraftingTemplate. Please keep an eye on this when launched and let me know. I worry it could become too burdensome to now have to document your fb on the issue and your fb on the testing in 2 different places.

I hear your concerns, but so long as communication in the artifact is limited to feedback on their testing process I think it should be manageable. But maybe we can add something to the backlog to review this after... maybe 5 staging reviews with the artifact?

**Comment by shiragoodman** [comment]
Thanks @briandeconinck ! per your responses above, I am going to create some follow up tickets (to be completed _after_ this ticket is completed). I'll update this comment with the new tickets once they're ready.

**Comment by briandeconinck** [comment]
@sterkenburgsara  @allison0034 Here's the big one: [Draft revision of "Prepare for an accessibility staging review"](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2432204801/47916+-+Prepare+for+an+accessibility+staging+review). For reference, here's the [currently published version](https://depo-platform-documentation.scrollhelp.site/developer-docs/prepare-for-an-accessibility-staging-review).

A few notes:
- This is roughly a 50/50 mix of new text I've written and existing text re-ordered and lightly edited. If you run into anything where there's a noticeable style shift, definitely feel free to edit or comment for me to edit.
- In the existing page, there's an advanced testing section that's very light on details, and then a few tests described elsewhere on the page that aren't grouped in with the foundational testing in one place, but then are grouped with them somewhere else. Since that's confusing (and I've never seen a VFS team do these tests), I've decided to put those additional tests in with advanced testing.
- I've also expanded the advanced testing section a bit with instructions from the internal [How to complete an accessibility review guidance](https://vfs.atlassian.net/wiki/spaces/PPS/pages/2059763776/How+to+complete+an+Accessibility+review+for+Staging+review+touchpoint) in order to make it more useful and actionable for VFS teams.
- I think this is organized in a way that makes sense? But I've also spent a lot of time with it and lost all sense of perspective.

Let me know if you have any questions! I'm going to start on all the other updates next, which should go pretty fast compared to this one.

**Comment by allison0034** [comment]
@briandeconinck I noticed you did not check off the tasks you have completed. Would you mind doing that? This si a lot and just trying to make sure I am reviewing the right stuff.

**Comment by briandeconinck** [comment]
@allison0034 Thanks for keeping me honest! Updated task list and checked things off to reflect current progress.

**Comment by shiragoodman** [comment]
@allison0034 could you let me know when you've completed your review? I want to take a look too but don't want to get in the way right now. Thank you!

**Comment by briandeconinck** [comment]
@shiragoodman Not sure how I never noticed this before, but there's an issue template in GitHub called ["Staging Review Accessibility Checklist."](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/.github/ISSUE_TEMPLATE/staging-review-accessibility-checklist.md)

Looking at the [commit history](https://github.com/department-of-veterans-affairs/va.gov-team/commits/master/.github/ISSUE_TEMPLATE/staging-review-accessibility-checklist.md), it looks like Mickin created it in 2020 and it received some light edits in February. From a quick search of past issues, I'm seeing 10 total instances of someone using this template --- but only two in the last year. Oddly enough, it never came up when I did my review of recent a11y artifacts submitted at staging. Those teams apparently used it as an internal checklist and didn't submit it as an artifact.

Since it's not getting much use and we're not going to point to it in our documentation, it's probably pretty harmless. But it's possible there might be confusion between this and the correct testing artifact. Is this something we want to retire?

**Comment by shiragoodman** [comment]
@briandeconinck I think I know the history here. Mickin use to be the Product Lead for the VSA teams (aka a subset of teams supporting account experience, benefits, health tools, etc.). He created artifacts like this one to help his teams out ([another one is the Collab Cycle initiative ticket](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/.github/ISSUE_TEMPLATE/collab-cycle-initiative.md)).

I'll reach out to Mickin to see if he has any concerns with us archiving these templates.

**Comment by briandeconinck** [comment]
Thank you!

**Comment by shiragoodman** [comment]
no issue from Mickin. Both have been deleted.

**Comment by allison0034** [comment]
My peer review is complete. I spoke with Brian and said he may want to consider changing some of the language when linking to documents and the titles. Otherwise looks great.

**Comment by briandeconinck** [comment]
@allison0034 and @sterkenburgsara I've now also completed a draft update to [How to complete an Accessibility review for Staging review touchpoint](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2438692865/47916+-+How+to+complete+an+Accessibility+review+for+Staging+review+touchpoint). The only changes are in Step 1 (new task for reviewing the artifact) and adding Step 11 (providing feedback on the artifact after the review).

Question for @shiragoodman -- [How to document feedback for Staging Review](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1839530098/How+to+document+feedback+for+Staging+Review) was on the list for impacted artifacts, but I don't see anything in there that I think needs to be edited. Did you have something specific in mind?

**Comment by allison0034** [comment]
@briandeconinck I made one comment about not commenting on the ticket, otherwise looks good to me!

**Comment by shiragoodman** [comment]
The WCAG 2.1 guidance page will be archived. There is nothing unique and/or relevant in the guidance.

**Comment by shiragoodman** [comment]
@briandeconinck sorry for the delay. I reviewed all updated docs and provided some comments. Most docs look good and require no change. Let me know if you have any questions/concerns!

**Comment by briandeconinck** [comment]
Current status:
- Artifact is live
- Platform website pages updated in Confluence and "Prepare for..." page moved to new home in the Collab Cycle section; waiting on Platform website publish cycle
- Internal documentation changes published

To-do once Platform website changes are published:
- Verify no broken links (especially from the artifact yaml form pointing to Platform website)
- Publish change history post
- Announce changes
- Breathe a sigh of relief

**Comment by shiragoodman** [comment]
so exciting! Thanks for all your hard work, @briandeconinck .
